google/data_source_google_service_account.go

```diff
@@ -0,0 +1,84 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceGoogleServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleServiceAccountRead,
+
+		Schema: map[string]*schema.Schema{
+			// Required
```

Comment:
The comments here aren't necessary since they duplicate the code below, and it's generally good practice not to include comments that duplicate code.  Thanks for putting the schema elements in our preferred order, though!

Reply:
No worries! I was trying to keep it similar to other datasource code, but I will remove it :)